mlir-tensorrt/compiler/include/mlir-tensorrt/Compiler/StableHloToExecutable.h

```diff
@@ -50,7 +51,7 @@ namespace mlirtrt::compiler {
 
 class StableHloToExecutableTask;
 
-struct StableHLOToExecutableOptions : public mlir::OptionsContext {
+struct StableHLOToExecutableOptions : public mlir::OptionsBundle<DebugOptions> {
```

Comment:
If we prefer not to change the inheritance hierarchy, then I can make the `OptionsBundle` a member instead (that's how I have it in the first commit).

Reply:
I've squashed the commits into one.